mesh/mesh.hpp

```diff
@@ -830,21 +837,31 @@ class Mesh
 
    const Element *GetFace(int i) const { return faces[i]; }
 
-   Geometry::Type GetFaceBaseGeometry(int i) const
+   Geometry::Type GetFaceGeometry(int i) const
    {
       return faces[i]->GetGeometryType();
    }
 
-   Geometry::Type GetElementBaseGeometry(int i) const
+   Geometry::Type GetElementGeometry(int i) const
    {
       return elements[i]->GetGeometryType();
    }
 
-   Geometry::Type GetBdrElementBaseGeometry(int i) const
+   Geometry::Type GetBdrElementGeometry(int i) const
    {
       return boundary[i]->GetGeometryType();
    }
 
+   // deprecated: "base geometry" no longer means anything
+   Geometry::Type GetFaceBaseGeometry(int i) const
+   { return GetFaceGeometry(i); }
+
+   Geometry::Type GetElementBaseGeometry(int i) const
+   { return GetElementGeometry(i); }
+
+   Geometry::Type GetBdrElementBaseGeometry(int i) const
+   { return GetBdrElementGeometry(i); }
```

Comment:
Should these be marked deprecated? I am guessing you didn't so as to not generate tons of warnings.

I think this is a good change since I never knew what the distinction between "base geometry" and "geometry" was.

Reply:
There used to be special refined Elements (e.g., `BisectedElement`) and the base geometry was the original geometry. This is long gone. I would like to use MFEM_DEPRECATED and change all occurrences to the simpler names, but it's a lot of changes that I felt would better be done in a separate PR.

Reply:
That sounds good to me 👍